src/components/line-graph.js

```diff
@@ -25,7 +28,13 @@ export function LineGraph(events, { width, height, title, start, end } = {}) {
     width,
     height,
     x: { type: 'utc', ticks: 'month', label: null },
-    y: { grid: true, inset: 10, label: 'RSR (%)', percent: true },
+    y: {
+      grid: true,
+      inset: 10,
+      label: 'RSR (%)',
+      percent: true,
+      zero: fromZero,
```

Comment:
Should we make "fromZero" the default for all charts that don't explicitly opt-out?

Reply:
See the network-wide stats that don't start the Y axis from zero.

<img width="1411" alt="Screenshot 2025-03-10 at 11 50 58" src="https://github.com/user-attachments/assets/7de5d3ec-8996-410e-abcf-b85a59076704" />

Reply:
See https://github.com/CheckerNetwork/spark-dashboard/pull/33